packages/gasket-cli/lib/index.d.ts

```diff
@@ -2,6 +2,11 @@ import type { GasketConfigDefinition, MaybeAsync } from '@gasket/engine';
 import type { PackageManager } from '@gasket/utils';
 import type { Command } from 'commander';
 
+/**
+ * https://godaddy-corp.atlassian.net/browse/PFX-572
```

Comment:
At least for create, the CLI does't play a part in anymore. So if you're going to make related type changes in this PR, move ownership of the types to `create-gasket-app`